src/index.html

```diff
@@ -5,8 +5,452 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#" 
+              class="nav__link" 
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+            Гаджети
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+            Планшеты
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Фото
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <div class="catalog__row">
+          <div class="card" data-qa="card">
+            <img 
+              src="images/imac.jpeg" 
+              alt="imac" 
+              class="card__img"
+            >
+      
+            <div class="card__title">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </div>
+      
+            <div class="card__cod">
+              Код товара: 195434
+            </div>
```

Comment:
```suggestion
            <h2 class="card__title">
              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
            </h2>
      
            <h3 class="card__cod">
              Код товара: 195434
            </h3>
```
краще використовувати семантичні теги